common/menu-list.c

```diff
@@ -209,10 +209,15 @@ int themeMenuScan(const char* target) {
     menuEntry_s* me = menuCreateEntry(ENTRY_TYPE_THEME);
 
     if(me) {
+<<<<<<< HEAD
```

Comment:
Merge conflicts...

Reply:
I resolved them on vs code... but for some reason  it didnt commit the first time... i had to add a newline and remove some spaces and then recommit for it to reflect the changed